qiskit/transpiler/passes/layout/disjoint_utils.py

```diff
@@ -55,7 +56,7 @@ def run_pass_over_connected_components(
             for qreg in dag.qregs:
                 out_dag.add_qreg(qreg)
             for creg in dag.cregs:
-                out_dag.add_cregs(creg)
+                out_dag.add_creg(creg)
```

Comment:
This looks like a typo since there is no `add_cregs` function

Reply:
yes. I agree.